core/src/main/java/org/infinispan/conflict/impl/ConflictMangerModuleLifecycle.java

```diff
@@ -0,0 +1,27 @@
+package org.infinispan.conflict.impl;
+
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.conflict.ConflictManager;
+import org.infinispan.factories.ComponentRegistry;
+import org.infinispan.lifecycle.AbstractModuleLifecycle;
+import org.infinispan.lifecycle.ModuleLifecycle;
+import org.kohsuke.MetaInfServices;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.1
+ */
+@SuppressWarnings("unused")
+@MetaInfServices(value = ModuleLifecycle.class)
+public class ConflictMangerModuleLifecycle extends AbstractModuleLifecycle {
```

Comment:
I didn't create an internal factory as the ConflictManager is only ever called internally from `ClusterCachestatus` which is created per cache inside the `ClusterTopologyManagerImpl` (this is global scope, so can't inject ConflictManager on per cache basis).

It was agreed that the ConflictManager should be exposed to the user via `ConflictManagerFactory.get(cache);`